plugins/mcreator-core/procedures/damagesource_isequalto.json

```diff
@@ -0,0 +1,25 @@
+{
+  "args0": [
+    {
+      "type": "field_data_list_selector",
+      "name": "damagesource",
+      "datalist": "damagesources",
+      "customEntryProviders": "damagesource"
```

Comment:
There's no damage type mod element (yet), I'd remove the `customEntryProviders` field